t/break-undo.vim

```diff
@@ -0,0 +1,125 @@
+" NB: MacVim defines several key mappings such as <D-v> by default.
+" The key mappings are defined from the core, not from any runtime file.
+" So that the key mappings are always defined even if Vim is invoked by
+" "vim -u NONE" etc.  Remove the kay mappings to ensure that there is no key
+" mappings, because some tests in this file assume such state.
+imapclear
+cmapclear
+
+let g:smartinput_break_undo = 1
```

Comment:
This line seems to do nothing, because all configurations will be reset for each test case.

Reply:
Oops, it's debug code 😢